graylog2-web-interface/src/views/components/searchbar/SearchBarForm.jsx

```diff
@@ -21,6 +22,16 @@ type Props = {
   children: ((props: FormikProps<Values>) => React$Node) | React$Node,
 };
 
+const validate = (values) => {
+  const errors = {};
+  if (values.timerange.type === 'absolute' && DateTime.isValidDateString(values.timerange.from) && values.timerange.from > values.timerange.to) {
```

Comment:
The biggest problem I faced while implementing these changes:

I had to check here if the "from" date is a valid date string, which is also part of the `AbsoluteTimeRangeSelector` validation . Imagine the following scenario:
Start date: 2020-06-18 11:04:34.82 (invalid date because last number is missing)
End date: 2020-06-10 11:04:34.823

In this case the error message would be "Start date must be before end date" instead of "Format must be: YYYY-MM-DD HH:mm:ss", without the mentioned date string check in the form validation.

At one point I've created a `SearchBarFormValidation` file, but it was way more complex than the current implementation and I really prefer the component based validation approach. Unfortunately I could not find a way to give the field validation a higher priority than the form validation.